maze_transformer/evaluation/plot_maze.py

```diff
@@ -66,20 +69,26 @@ class StyledPath(PathFormat):
 
 
 def process_path_input(
-    path: CoordArray | StyledPath,
+    path: CoordList | CoordArray | StyledPath,
```

Comment:
Can we unify CoordList and CoordArray? Any reason why we need both?

Reply:
Very much agree that these should be unified! Beyond the scope of this PR but tackling it in the next one